plutus-pab/tx-inject/start-all-servers.sh

```diff
@@ -0,0 +1,11 @@
+rm -f ./primary.db
+stack exec plutus-scb -- --config=./config.yaml migrate
```

Comment:
`plutus-pab`

Let's try to integrate this into the main `plutus-pab` executable at some point (I'll make an issue)